app/assets/stylesheets/application.scss

```diff
@@ -10,3 +10,7 @@
 // Your CSS partials
 @import "components/index";
 @import "pages/index";
+
+body {
```

Comment:
Body margin was defaulting to 8px;

Overwritten in main application SCSS styling to clearly display that margin for body is now 0px.